### PR TITLE
Package builder for Serialbox2

### PR DESCRIPTION
**Pull request by clementval**

### CODE REVIEW

**Comment by clementval** [comment]
@lxavier Tested with GNU on arolla

File: env.arolla.sh [hunk 1]
```diff
@@ -305,6 +310,7 @@ EOF
         cat > $ENVIRONMENT_TEMPFILE <<-EOF
             # Generated with the build script
             # implicit module purge
+            module load cmake/3.14.3
```
Comment (lxavier) [inline env.arolla.sh]: this is in the COSMO_TESTENV section - for tsa with RH7.6 - module should be 
cmake/3.14.5

File: env.arolla.sh [hunk 2]
```diff
@@ -149,6 +150,7 @@ else
         module use /apps/arolla/UES/jenkins/RH7.6/generic/easybuild/modules/all
         # Generated with the build script
         # implicit module purge
+        module load cmake/3.14.3
```
Comment (lxavier) [inline env.arolla.sh]: this is in the COSMO_TESTENV section - for tsa with RH7.6 - module should be
cmake/3.14.5

File: env.arolla.sh (hunk 1 above)
Comment (clementval) [reply env.arolla.sh]: I don't see any reference to `tsa` or a `env.tsa.sh` ...

Comment (lxavier) [reply env.arolla.sh]: you need to do
       module unuse /apps/arolla/UES/modulefiles
        module use /apps/arolla/UES/jenkins/RH7.6/generic/easybuild/modules/all
to see the correct modules on tsa

Comment (clementval) [reply env.arolla.sh]: Ok but there is still no `env.tsa.sh` ... so is there a dirty trick somewhere and it loads the same env file as arolla?

**Comment by lxavier** [comment]
Yes very dirty trick,

export COSMO_TESTENV

**Comment by clementval** [comment]
@lxavier Update done ... sorry for the delay :-(

**Comment by lxavier** [comment]
thanks !

**Pull request merged**